Mathlib/Data/Matroid/IndepAxioms.lean

```diff
@@ -273,20 +263,22 @@ theorem _root_.Matroid.existsMaximalSubsetProperty_of_bdd {P : Set α → Prop}
     rintro x ⟨Y, ⟨hY,-,-⟩, rfl⟩
     obtain ⟨n₀, heq, hle⟩ := hP Y hY
     rwa [ncard_def, heq, ENat.toNat_coe]
-    -- have := (hP Y hY).2
-  obtain ⟨Y, hY, hY'⟩ := Finite.exists_maximal_wrt' ncard _ hfin ⟨I, hI, rfl.subset, hIX⟩
-  refine ⟨Y, hY, fun J ⟨hJ, hIJ, hJX⟩ (hYJ : Y ⊆ J) ↦ (?_ : J ⊆ Y)⟩
-  have hJfin := finite_of_encard_le_coe (hP J hJ)
-  refine (eq_of_subset_of_ncard_le hYJ ?_ hJfin).symm.subset
-  rw [hY' J ⟨hJ, hIJ, hJX⟩ (ncard_le_ncard hYJ hJfin)]
+  obtain ⟨Y, ⟨hY, hIY, hYX⟩, hY'⟩ :=
+    Finite.exists_maximal_wrt' ncard _ hfin ⟨I, hI, rfl.subset, hIX⟩
+
```

Comment:
I've tried to revert to the way things were. I tend to include blank lines to separate ideas in proofs, and when I was rewriting to work with the new API, I didn't faithfully preserve where the blanks were.

Reply:
@apnelson1 if you feel there is a separate enough flow ‌/ idea in the proof which is not otherwise captured by the structure of the proof, that's a sign that you should add a *comment* instead of a blank line.